src/javascript/ContentManager/ContentManager.utils.test.js

```diff
@@ -0,0 +1,27 @@
+import {removeFileExtension, getNewCounter} from './ContentManager.utils';
+
+describe('zip utils', () => {
```

Comment:
You should use one 'describe' per function to test, with one or many 'it' within.